### ISSUE TITLE
Strategy for automated backup of OSB server contents

### ISSUE THREAD

**Issue by pgleeson** [OpenSourceBrain/redmine#188]
Some of the previous scripts I used to backup the database don't work on AWS. Need some updated way to pull down & archive db.

**Comment by adrianq** [comment]
Created two scripts:
- [Create a db in the amazon instance](https://github.com/OpenSourceBrain/redmine/blob/development/script/databaseBackup)
- [Download db dump to local](https://github.com/OpenSourceBrain/redmine/blob/development/script/downloadBackupFromEC2)

Both of them content some instructions about how to create a crontab. Should be straight-forward...

I have created a cron job in dev and prod which is executed every day at night. I have also configured my laptop to download the backups.
